webv2/session/session_service.go

```diff
@@ -7,7 +7,7 @@ import (
 	"cloud.google.com/go/spanner"
 	database "cloud.google.com/go/spanner/admin/database/apiv1"
 	"cloud.google.com/go/spanner/admin/database/apiv1/databasepb"
-	"github.com/GoogleCloudPlatform/spanner-migration-tool/conversion"
+	spanneraccessor "github.com/GoogleCloudPlatform/spanner-migration-tool/accessors/spanner"
```

Comment:
can we make this camel case, for better readability?

Reply:
The golang style guides is totally against camelcase and capital letters, they only prefer small letter for package names. The recommendation is to do it like this and if necessary, use snake case ('_' separated).

Amongst direct appending and snake case, the former seems to be preferred. [LINK](https://yaqs.corp.google.com/eng/q/2393822631133446144)